content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -166,15 +166,15 @@ Einer unserer Schwerpunkte ist der Kampf gegen die sexuelle Ausbeutung und den M
 
 ### Die Matrix Community
 
-Die breitere Matrix-Community hat starke Initiativen zur Missbrauchsbekämpfung, die uns sehr ermutigen. Insbesondere die Arbeit am [Draupnir-Moderationsbot-Projekt](https://github.com/the-draupnir-project/Draupnir) und der Community Moderation Effort ([CME](https://matrix.to/#/#community-moderation-effort-bl:neko.dev)) sind hervorragende Ergänzungen zum Sicherheitsökosystem von Matrix. Auf der Ebene der Server-Tools sind das [Synapse-Admin-Tool](https://github.com/Awesome-Technologies/synapse-admin) von Awesome Technologies und der [Fork von etke.cc](https://github.com/etkecc/synapse-admin) großartige Beispiele dafür, wie Open-Source-Ökosysteme zu Sicherheits-Tools beitragen können. Wir freuen uns über ihre Arbeit und hoffen, dass wir in den kommenden Monaten und Jahren weitere von der Basis ausgehende Entwicklungen sehen werden. 
+Die breitere Matrix-Community hat starke Initiativen zur Missbrauchsbekämpfung, die uns sehr ermutigen. Insbesondere die Arbeit am [Draupnir-Moderationsbot-Projekt](https://github.com/the-draupnir-project/Draupnir) und der Community Moderation Effort ([CME](https://matrix.to/#/#community-moderation-effort-bl:neko.dev)) sind hervorragende Ergänzungen zum Schutzökosystem von Matrix. Auf der Ebene der Server-Werkzeuge sind [Synapse-Admin](https://github.com/Awesome-Technologies/synapse-admin) von Awesome Technologies und der [Fork von etke.cc](https://github.com/etkecc/synapse-admin) großartige Beispiele dafür, wie Open-Source-Ökosysteme zu Schutzwerkzeugen beitragen können. Wir freuen uns über ihre Arbeit und hoffen, dass wir in den kommenden Monaten und Jahren weitere auf eigenen Füßen stehende Entwicklungen sehen werden.
```

Comment:
I don't think the "Basis" formulation works and the idiom "auf eigenen Füßen" is a better fit